bindata/network/ovn-kubernetes/self-hosted/004-config.yaml

```diff
@@ -58,6 +58,9 @@ data:
 {{- if .DNS_NAME_RESOLVER_ENABLE }}
     enable-dns-name-resolver=true
 {{- end }}
+{{- if .OVN_PERSISTENT_IPS_ENABLE }}
+    enable-persistent-ips=true
+{{- end }}
```

Comment:
If we pass the option as parameter (which we need to trigger a rollout) then I don't think we usually add the option in the config map so I wouldn't do it here either.

Reply:
Done.